mediapipe/calculators/ovms/modelapiovmssessioncalculator.cc

```diff
@@ -154,10 +154,17 @@ class ModelAPISessionCalculator : public CalculatorBase {
             if (!isServerReady) {
                 REPORT_CAPI_STATUS_NULL(OVMS_ServerStartFromConfigurationFile(cserver, _serverSettings, _modelsSettings));
             }
-            while (!isServerReady) {
-                std::this_thread::sleep_for(std::chrono::milliseconds(10));
+            
+            int timeoutCounter = 100;
```

Comment:
timeout for readiness is not a good option because loading time depends on the model.